plasmapy/formulary/parameters.py

```diff
@@ -1769,6 +1769,16 @@ def upper_hybrid_frequency(B: u.T, n_e: u.m ** -3) -> u.rad / u.s:
     where :math:`ω_{ce}` is the electron gyrofrequency and
     :math:`ω_{pe}` is the electron plasma frequency.
 
+    The upper hybrid frequency is a resonance for electromagnetic
+    waves in magnetized plasmas, namely for the X-mode. These are
+    waves with their wave electric field being perpendicular to 
+    the background magnetic field. In the cold plasma model, i.e.
+    without any finite temperature effects, the resonance acts
+    merely as a resonance such that power can be deposited there. 
+    If finite temperature effects are considered, mode conversion
+    can occur at the upper hybrid resonance, coupling to the 
+    electrostatic electron Bernstein wave. 
+
```

Comment:
```suggestion
    The upper hybrid frequency is a resonance for electromagnetic
    waves in magnetized plasmas, namely for the X-mode. These are
    waves with their wave electric field being perpendicular to
    the background magnetic field. In the cold plasma model, i.e.
    without any finite temperature effects, the resonance acts
    merely as a resonance such that power can be deposited there.
    If finite temperature effects are considered, mode conversion
    can occur at the upper hybrid resonance, coupling to the
    electrostatic electron Bernstein wave.

```
A minor suggestion to remove trailing white space.